package.json

```diff
@@ -10,9 +10,8 @@
     "flow.copy:src": "./node_modules/.bin/cpy ng2-select.ts \"components/*.ts\" ts --parents",
     "flow.clean": "./node_modules/.bin/del bundles coverage demo-build typings \"components/**/*.+(js|d.ts|js.map)\" dist \"ng2-select.+(js|d.ts|js.map)\"",
     "flow.deploy:gh-pages": "npm run flow.build:prod && ./node_modules/.bin/gh-pages -d demo-build",
-    "flow.eslint": "./node_modules/.bin/eslint --ignore-path .gitignore  --ext js --fix . .config",
-    "flow.tslint": "./node_modules/.bin/gulp lint",
-    "flow.lint": "npm run flow.eslint && npm run flow.tslint",
+    "flow.eslint": "./node_modules/.bin/eslint --ignore-path .eslintignore  --ext js --fix . .config",
+    "flow.lint": "npm run flow.eslint",
```

Comment:
There are 2 changes here.
- .gitignore and .eslintignore are fundamentally different things
- remove use of gulp ts lint in favour of webpack ts linting,

I actually think it would be easier to use just webpack for almost everything, but maybe that's just personal preference